arch/arm/src/samd5e5/sam_usb.c

```diff
@@ -8566,10 +8531,10 @@ static inline void sam_sw_initialize(struct sam_usbhost_s *priv)
 
   sam_reset_pipes(priv, false);
 
-  /* Initialize semaphores */
+  /* Initialize semaphore & mutex */
```

Comment:
In some places `&` is used and in other `and`. So maybe we can use similar style in all the places?

Reply:
after search, all place use &.